kafka-connector/src/main/java/com/google/pubsub/kafka/source/CloudPubSubGRPCSubscriber.java

```diff
@@ -41,9 +41,11 @@
   private GrpcSubscriberStub subscriber;
   private Random rand = new Random(System.currentTimeMillis());
   private CredentialsProvider gcpCredentialsProvider;
+  private String endpoint;
 
-  CloudPubSubGRPCSubscriber(CredentialsProvider gcpCredentialsProvider) {
+  CloudPubSubGRPCSubscriber(CredentialsProvider gcpCredentialsProvider, String endpoint) {
```

Comment:
nit. @Nullable since this is rare for strings, or consider making "pubsub.googleapis.com" the default flag value and not thinking about null?

Reply:
Went with a default flag value.